### PR TITLE
Fixes broken link of contributor cheat sheet

### LINKED ISSUE #218: Broken Link of `contributor cheat sheet` needs to fix
## Bug Report

I have observed same kind of issue in various kubernetes-csi project.
this happens because after the localization there are too much modifications done in the various directories.
I have observed same issue in [this ](https://github.com/kubernetes-csi/csi-proxy/blob/master/CONTRIBUTING.md) page also.

It has one broken link of the `contributes  cheat sheet` which needs to fix.
I will try to look in further csi repo as well and try to fix it as soon as I can

/kind bug
/assign

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>
Fixes #218

### CODE REVIEW

**Comment by amolmote** [comment]
@ddebroy please provide the release-note-lable now this PR is ready to merge

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 
 - [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
 - [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
-- [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet.md) - Common resources for existing developers
+- [Contributor Cheat Sheet](https://github.com/kubernetes/community/blob/master/contributors/guide/contributor-cheatsheet/README.md) - Common resources for existing developers
```
Comment (mauriciopoppe) [inline CONTRIBUTING.md:17]: I'd have used `https://git.k8s.io/` instead e.g. https://git.k8s.io/community/contributors/guide/contributor-cheatsheet/README.md

**Pull request merged**